### PR TITLE
Add network selector

### PR DESCRIPTION
**Pull request by denkravchu**
### Implemented:

- New Network Select block design (https://www.figma.com/design/Bw2OVsNb9cRtP8QGUvV803/NETWORK-SELECTOR?node-id=0-1&t=K62qKZRV8PvKfGyT-1)
- Made Hover effects ease n smooth

### Moments:

- Used website blue color tone instead of the one was used in design too keep the style clean
- Used website font family instead of the one was used in design to keep the style clean

Screenshot 2024-07-15 at 18 26 18

### CODE REVIEW

**Comment by pshenmic** [comment]
Maybe we can make animation woks only at clicking? It looks a bit confusing, when this animations works for hover and click at the same time

**Comment by pshenmic** [comment]
Also, let's remove devnet-abc for now, let's leave only mainnet and testnet

**Comment by denkravchu** [comment]
> Maybe we can make animation woks only at clicking? It looks a bit confusing, when this animations works for hover and click at the same time

In my opinion the click is less convenient then hover in current situation, however maybe we really should leave only one way of toggling. Up to you then:)

File: packages/frontend/src/components/navbar/NetworkSelect.scss [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        cursor: pointer;
+        font-weight: 700;
+        font-size: 12px;
+        letter-spacing: 0.04em;
+        display: flex;
+        align-items: center;
+        gap: 10px;
+        z-index: 11;
+
+
```
Comment (alexeyandreevsky) [inline packages/frontend/src/components/navbar/NetworkSelect.scss:31]: Excess empty line

File: packages/frontend/src/components/navbar/NetworkSelect.scss [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            stroke-width: 1.50px;
+            transform: rotate(180deg);
+            transition: transform ease 0.5s;
+        }
+        .Arrow__Active{
+            transform: rotate(0deg);
+            transition: transform ease 0.5s;
+        }
+    }
+
```
Comment (alexeyandreevsky) [inline packages/frontend/src/components/navbar/NetworkSelect.scss:43]: Excess empty line

File: packages/frontend/src/components/navbar/NetworkSelect.scss [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+.NetworkSelect__DropdownWrapper{
+    position: absolute;
+    top: calc(5px + 100%);
+    right: 0;
+    z-index: 10;
+    opacity: 0;
+    visibility: hidden;
+    pointer-events: none;
+    transition: ease 0.5s;
+    &::before{
```
Comment (alexeyandreevsky) [inline packages/frontend/src/components/navbar/NetworkSelect.scss:68]: Need to add line breaks before nested blocks

**Comment by riongull** [comment]
Will "Local" be an option?

**Comment by pshenmic** [comment]
@riongull no, it is too complicated to get this done on the PE's side, but you can setup your own local version of PE connected to the local network

File: packages/frontend/package.json [hunk 4]
```diff
@@ -12,6 +12,7 @@
     "@testing-library/user-event": "^13.5.0",
     "d3": "^7.9.0",
     "express": "^4.18.2",
+    "framer-motion": "^11.3.3",
```
Comment (alexeyandreevsky) [inline packages/frontend/package.json:15]: Please remove unused module

File: packages/frontend/src/components/navbar/Navbar.js [hunk 5]
```diff
@@ -50,45 +53,50 @@ function Navbar () {
...
-            size={'md'}
-            icon={isOpen ? <CloseIcon /> : <HamburgerIcon />}
-            aria-label={'Open Menu'}
-            display={{ lg: 'none' }}
-            onClick={isOpen ? onClose : onOpen}
-          />
+      <Flex
+        className={'Navbar'}
+        maxW='1980px'
+        ml='auto'
```
Comment (alexeyandreevsky) [inline packages/frontend/src/components/navbar/Navbar.js:59]: Wrap the component parameters in {}, please.
`ml={'auto'}` etc.

File: packages/frontend/src/components/navbar/NetworkSelect.scss [hunk 6]
```diff
@@ -0,0 +1,84 @@
+
+.NetworkSelect {
+    position: relative;
+    display: flex;
+    align-items: center;
+    gap: 8px;
+    
+    >p {
```
Comment (alexeyandreevsky) [inline packages/frontend/src/components/navbar/NetworkSelect.scss:8]: Space please: `> p {`

File: packages/frontend/src/components/ui/NavigationButton/NavigationButton.scss [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+        align-items: center;
+        justify-content: center;
+        font-weight: 400;
+        font-size: 10px;
+        letter-spacing: 0.04em;
+        transition: ease 0.5s;
+    }
+}
+
+.NavigationButton__NoActive{
```
Comment (alexeyandreevsky) [inline packages/frontend/src/components/ui/NavigationButton/NavigationButton.scss:43]: Space before `{` please

File: packages/frontend/src/components/navbar/NetworkSelect.scss [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+    border-top: 4px solid #52585d;
+    border-radius: 10px;
+    padding: 8px 4px;
+    background: #24282b;
+    display: flex;
+    flex-direction: column;
+    gap: 4px;
+}
+
+.NetworkSelect__DropdownWrapper{
```
Comment (alexeyandreevsky) [inline packages/frontend/src/components/navbar/NetworkSelect.scss:59]: Space before `{`

File: packages/frontend/src/components/navbar/NetworkSelect.js [hunk 9]
```diff
@@ -0,0 +1,43 @@
+'use client'
+
+import { useState } from 'react'
+import './NetworkSelect.scss'
+import Dropdown from './Dropdown'
+
+const listNetworks = [
+  { name: 'testnet', subname: '', activeButton: true },
+  { name: 'mainnet', subname: 'Soon', activeButton: false }
+  // { name: 'devnet-abc', subname: 'Soon', activeButton: false }
```
Comment (alexeyandreevsky) [inline packages/frontend/src/components/navbar/NetworkSelect.js:10]: Remove comment

File: packages/frontend/src/components/ui/NavigationButton/NavigationButton.scss [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+    padding: 12px 20px;
+    background-color: transparent;
+    border-radius: 10px;
+    white-space: nowrap;
+    font-weight: 700;
+    font-size: 12px;
+    letter-spacing: 0.04em;
+    transition: ease 0.5s;
+   
+    &:hover{
```
Comment (alexeyandreevsky) [inline packages/frontend/src/components/ui/NavigationButton/NavigationButton.scss:21]: Space before `{`

File: packages/frontend/src/components/ui/NavigationButton/NavigationButton.scss [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+        background-color: rgba(82, 88, 93, 0.43);
+
+        .NavigationButton__SubName{
```
Comment (alexeyandreevsky) [inline packages/frontend/src/components/ui/NavigationButton/NavigationButton.scss:24]: Space

File: packages/frontend/src/components/navbar/NetworkSelect.scss [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+        content: '';
+        position: absolute;
+        left: 0;
+        top: -50%;
+        width: 100%;
+        height: 100%;
+    }
+}
+
+.DropdownWrapper__Active{
```
Comment (alexeyandreevsky) [inline packages/frontend/src/components/navbar/NetworkSelect.scss:79]: Space before `{`

File: packages/frontend/src/components/navbar/NetworkSelect.scss [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+        z-index: 11;
+
+        .NetworkSelect__Arrow {
+            color: var(--chakra-colors-white-50);
+            stroke-width: 1.50px;
+            transform: rotate(180deg);
+            transition: transform ease 0.5s;
+        }
+
+        .Arrow__Active{
```
Comment (alexeyandreevsky) [inline packages/frontend/src/components/navbar/NetworkSelect.scss:39]: Space before `{` please. In other places in the file too.

**Review by alexeyandreevsky — COMMENTED** [review]
Good job. Please add some code style changes.

File: packages/frontend/src/components/navbar/Navbar.js (hunk 5 above)
Comment (pshenmic) [reply packages/frontend/src/components/navbar/Navbar.js:59]: Is there a lint rule for that?

**Review by alexeyandreevsky — DISMISSED** [review]
Nice

**Review by alexeyandreevsky — APPROVED** [review]

**Pull request merged**